microservices/submission-service/src/replay-submission/replay-submission-crud/replay-submission-crud.controller.ts

```diff
@@ -15,4 +15,11 @@ export class ReplaySubmissionCrudController {
         await this.crudService.removeSubmission(data.submissionId);
         return {removed: true};
     }
+
+    @MessagePattern(SubmissionEndpoint.GetSubmissionRejections)
+    async getSubmissionRejections(@Payload() payload: unknown): Promise<IGetSubmissionRejections_Response> {
```

Comment:
I think there is a way to do this, without needing to create an interface; but I could be wrong. Lets look into it on Sunday.

Reply:
`SubmissionOutput<SubmissionEndpoint.GetSubmissionRejections>`